### PR TITLE
feat(ecosystems): support Red Hat fully

### PR DESCRIPTION
**Pull request by andrewpollock**
Red Hat records are not importing as intended, I believe this will correct that.

### CODE REVIEW

File: osv/ecosystems/redhat.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Red Hat Linux ecosystem helper."""
+
+from ..third_party.univers.rpm import RpmVersion
+from .helper_base import Ecosystem
+
+
+class Redhat(Ecosystem):
```
Comment (oliverchang) [inline osv/ecosystems/redhat.py:20]: nit: RedHat for proper casing?

**Review by oliverchang — APPROVED** [review]

File: osv/ecosystems/redhat.py (hunk 1 above)
Comment (andrewpollock) [reply osv/ecosystems/redhat.py]: Addressed in 4aecf99

File: osv/ecosystems/redhat.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Red Hat Linux ecosystem helper."""
+
+from ..third_party.univers.rpm import RpmVersion
+from .helper_base import Ecosystem
+
+
+class RedHat(Ecosystem):
+  """"Red Hat Linux ecosystem"""
```
Comment (oliverchang) [inline osv/ecosystems/redhat.py:21]: nit: extra quote in """"

File: osv/ecosystems/redhat.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+  @property
+  def name(self):
+    return "Red Hat"
```
Comment (oliverchang) [inline osv/ecosystems/redhat.py:25]: nit: single quotes

**Review by oliverchang — APPROVED** [review]

File: osv/ecosystems/redhat.py (hunk 2 above)
Comment (andrewpollock) [reply osv/ecosystems/redhat.py:21]: Addressed in 430da5fc (which also fixes the origin)

File: osv/ecosystems/redhat.py (hunk 3 above)
Comment (andrewpollock) [reply osv/ecosystems/redhat.py:25]: Addressed in 430da5fc (which also fixes the origin)

**Pull request merged**